src/WebJobs.Extensions.Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -263,6 +262,15 @@ internal static void ApplyBindingData(object target, IDictionary<string, object>
                 var bindingData = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
                 if (request.Content != null && request.Content.Headers.ContentLength > 0)
                 {
+                    if (request.Content.Headers.ContentType != null)
+                    {
+                        var charSet = request.Content.Headers.ContentType.CharSet;
+                        if (!string.IsNullOrEmpty(charSet))
```

Comment:
I wonder why you do this fixup here, rather than in an early top level piece of middleware (e.g. like [this](https://github.com/Azure/azure-functions-host/blob/02f70ce1e5ed67faa9587c5bcec539a5cc9d1b6c/src/WebJobs.Script.WebHost/Middleware/AppServiceHeaderFixupMiddleware.cs#L25)). If we're working around a bug, I'd say we want to do this early in an isolated location so ALL downstream consumers get the fix.

Reply:
Also, once the .NET Core bug gets fixed, we'll want to remove this, right? So can you put a TEMP link in the source pointing to the issue?